addons/mrp/models/mrp_production.py

```diff
@@ -238,6 +238,7 @@ def _compute_confirm_cancel(self):
                 productions_with_done_move[production_record[0]] = True
         for production in self:
             production.confirm_cancel = productions_with_done_move.get(production.id, False)
+    description = fields.Char()
```

Comment:
can you move this definition with the other fields ?
it's also missing an help

Reply:
done